### PR TITLE
Sim visualization cleanup

### PR DESCRIPTION
**Pull request by peabody124**
Small cleanup of the simulation LED output and code based on
backporting https://github.com/d-ronin/dRonin/pull/173/ and https://github.com/d-ronin/dRonin/pull/353/

Tested and produces this output:

```
(next)$ ../build/sim_posix/sim_posix.elf 
tcp dev 0 - socket 4 opened - result 0
PIOS: LEDs: [.*]
PIOS: LEDs: [**]
PIOS: LEDs: [*.]
PIOS: LEDs: [**]
PIOS: LEDs: [*.]
```

### CODE REVIEW

File: flight/PiOS.posix/posix/pios_led.c [hunk 1]
```diff
@@ -28,19 +28,53 @@
...
+
+			mask <<= 1;
+		}
+
+		leds[PIOS_LED_NUM] = 0;
+
+		printf("PIOS: LEDs: [%s]\n", leds);
+	}
+
+	//printf("PIOS: LED %i status %i\n",led,stat);
```
Comment (kubark42) [inline flight/PiOS.posix/posix/pios_led.c]: Can't see any reason for this comment. @mlyle  proposed it was good to leave it in, but I thought his argument was weak. Can we get rid of it, as it's not helpful? If the intention is to be able to turn it on for debugging purposes, a #define would be more appropriate.

File: flight/PiOS.posix/posix/pios_led.c [hunk 2]
```diff
@@ -28,19 +28,53 @@
...
 
 /* Private Function Prototypes */
 
+DONT_BUILD_IF(PIOS_LED_NUM > 8, piosLedNumTooHigh);
 
 /* Local Variables */
-static uint8_t LED_GPIO[PIOS_LED_NUM];
+static uint8_t ledStatus;
 
+static inline void PIOS_SetLED(uint32_t led, uint8_t stat) {
```
Comment (kubark42) [inline flight/PiOS.posix/posix/pios_led.c]: `{` on newline.

**Comment by kubark42** [comment]
LGTM aside from those two style issues.

**Comment by kubark42** [comment]
@peabody124 want to fix those couple issues and I'll merge this? I think this is the last no-brainer before we're done with #2127 and we'll wrap it up with #2172.

**Comment by peabody124** [comment]
@kubark42 done

**Pull request merged**